java-frontend/src/main/java/org/sonar/java/UCFGJavaVisitor.java

```diff
@@ -224,6 +227,9 @@ private void buildCall(Tree element, UCFGBuilder.BlockBuilder blockBuilder, Iden
     if (element.is(METHOD_INVOCATION)) {
       MethodInvocationTree methodInvocationTree = (MethodInvocationTree) element;
       buildMethodInvocation(blockBuilder, idGenerator, methodInvocationTree);
+    } else if (element.is(NEW_CLASS)) {
```

Comment:
What about method references on constructors, such as `MyStringsList.map(MySink::new).forEach(...)` ?
(I know this is also probably not handled for simple method references)

Reply:
As we don't handle collections I don't think this is relevant and linked to this PR.